### DISCUSSION TITLE
Why do variables have different names for Get, Create and Update functions?

### DISCUSSION THREAD

**Discussion by eudjinnl** [netbox-community/netbox#7578] [Q&A]
For example, if I want to create an ip address, I use this:
ip = netbox.ipam.create_ip_address(address = "192.168.1.110/24", tenant = 9, description = "test")

but if I want to check existance of the ip before creation I have to change variable's names like this:

ip = netbox.ipam.get_ip_addresses(address= "192.168.1.110/24", **tenant_id** = 9, description = "test")

if I want to update this ip, I have to change names again:

ip = netbox.ipam.update_ip(**ip_address**="192.168.1.110/24", **tenant**=9, description = "test1")

It looks so inconvenient for me but maybe there is some idea behind it?

Are there any ways to do there operations easier?

**Comment by DanSheps** [comment]
This is related to the way the filters are named in the rest API.

I suppose we could audit all the models and see where we could improve

**Reply by eudjinnl** [reply]
Oh. I understand.
Thank you. Will be waiting for updates and thank you for the great product.
